vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_tables.scss

```diff
@@ -0,0 +1,48 @@
+// Table styles for SUTD Project
+.sutd-table{
```

Comment:
@GracielaCesareo Since tables are a new element in the styleguide, maybe this class can have a more generic name like `styleguide-table`. That won't affect any other project that is currently using the styleguide. WDYT?

Reply:
@lubc You right. I've modified the class